Revolution Rotation Calculator.py

```diff
@@ -34,33 +34,30 @@ def AbilityRotation(Permutation, AttackSpeed, Activate_Bleeds, Gain, Start_Adren
     Shards = 0
     Adrenaline = Start_Adrenaline
     # --- Calculations begin here --- #
-    AbilityPath.append('AUTO' + ' D: ' + str(round(Current,1)) + ' T: ' + str(round(Clock, 1)) + ' A: ' + str(Adrenaline))
+    AbilityPath.append(f'AUTO D: {round(Current, 1)} T: {round(Clock, 1)} A: {Adrenaline}')
     Current += 50
     Adrenaline += Auto_Adrenaline
-    if Adrenaline > 100:
-            Adrenaline = 100
-    if Adrenaline < 0:
+    if Adrenaline >= 100:
+        Adrenaline = 100
+        for Ability in UltimateIterator:

```

Comment:
@NightShadeI I did not attempt to change the functionality of the original code, but I think there's possibly a bug here. If Adrenaline >= 100, the threshold ability should be set to true too!

Reply:
Good spot - I'll get around to changing many of these small things later (thanks a lot 👍).